spec/integration/provider/mount_spec.rb

```diff
@@ -32,7 +32,7 @@ def create_fake_fstab(initially_contains_entry)
       when %r{/s?bin/mount}
         if command.length == 1
           if @mounted
-            "#{@current_device} on /Volumes/foo_disk (msdos, #{@current_options})\n"
+            "#{@current_device} on /Volumes/foo_disk type msdos (msdos, #{@current_options})\n"
```

Comment:
This was given OS X/Darwin type mount output, but Facter was stubbed with Linux/RedHat. I changed the mount output to something Linux-compatible.